clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/PersistenceDAO.scala

```diff
@@ -1,25 +1,47 @@
 package org.broadinstitute.clio.server.dataaccess
 
+import java.io.File
+
 import org.broadinstitute.clio.server.ClioServerConfig
 import org.broadinstitute.clio.server.ClioServerConfig.Persistence
 import org.broadinstitute.clio.server.dataaccess.elasticsearch.{
   ClioDocument,
   ElasticsearchIndex
 }
-
 import com.sksamuel.elastic4s.Indexable
 import com.typesafe.scalalogging.LazyLogging
 
 import scala.concurrent.{ExecutionContext, Future}
-
 import java.nio.file.{Files, Path}
+import java.util.UUID
+
+import akka.actor.ActorSystem
+import akka.stream.{ActorMaterializer, ActorMaterializerSettings, Supervision}
+import akka.stream.alpakka.file.scaladsl.Directory
+import akka.stream.scaladsl.{Flow, Sink, Source}
+import io.circe.Decoder
+import io.circe.parser._
 
 /**
   * Persists metadata updates to a source of truth, allowing
   * for playback during disaster recovery.
   */
 trait PersistenceDAO extends LazyLogging {
 
+  private implicit val system = ActorSystem("clio")
```

Comment:
We've [already got an ActorSystem](https://github.com/broadinstitute/clio/blob/develop/clio-server/src/main/scala/org/broadinstitute/clio/server/ClioServer.scala#L22) running the server, you shouldn't need to create another one here. There's an existing materializer too.

I'd either change `PersistenceDAO` to be an abstract class taking an implicit materializer, or add an implicit materializer parameter to the new method.

Reply:
Tried the abstract class idea yesterday but got an assertion failure I couldn't figure out.
Adding another implicit to the methods works though.

Reply:
Unfortunately, I added a materializer to the Spec class.
Scalatest must have one, or at least an ActorSystem, but I don't know how to get at it.
Maybe you do?

Reply:
If you make the spec extends `TestKit(ActorSystem("some-name")) with AsyncFlatSpecLike` instead of `AsyncFlatSpec`, you'll get an actor system.

Reply:
You'll also want to throw a `with BeforeAndAfterAll` on there, with:
```scala
override def afterAll(): Unit = {
    shutdown()
    super.afterAll()
  }
```
to make sure the actor system dies properly.